content/course/interchain-token-transfer/07-tokens-on-multiple-chains/01-tokens-on-multiple-chain.mdx

```diff
@@ -1,10 +1,20 @@
 ---
-title: Tokens on Multiple Chains
-description: TBD
+title: Overview of Multi-hop Transfers
+description: Learn about how ICTT supports multi-hop transfers between spoke chains.
 updated: 2024-05-31
-authors: [ashucoder9]
+authors: [owenwahlgren]
 icon: Book
 ---
 
 The token bridge also supports "multi-hop" transfers, where tokens can be transferred between spoke chains. To illustrate, consider two spokes _S<sub>a</sub>_ and _S<sub>b</sub>_ that are both connected to the same remote _H_. A multi-hop transfer from _S<sub>a</sub>_ to _S<sub>b</sub>_ first gets routed from _S<sub>a</sub>_ to _H_, where spoke balances are updated, and then _H_ automatically routes the transfer on to _S<sub>b</sub>_.
```

Comment:
```suggestion
The token bridge also supports "multi-hop" transfers, where tokens can be transferred between remote chains via the home chain. To illustrate, consider two remotes _R<sub>a</sub>_ and _R<sub>b</sub>_ that are both connected to the same home _H_. A multi-hop transfer from _R<sub>a</sub>_ to _R<sub>b</sub>_ first gets routed from _R<sub>a</sub>_ to _H_, where the remote balances are updated, and then _H_ automatically routes the transfer on to _R<sub>b</sub>_.
```

We've renamed ICTT from Hub and Spoke to Home and Remote, so we want to avoid using Spoke and stick with home and remote